WalletWasabi.Fluent/ViewModels/Wallets/Send/AdvancedSendOptionsViewModel.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Globalization;
+using System.Reactive.Linq;
+using NBitcoin;
+using ReactiveUI;
+using WalletWasabi.Fluent.Validation;
+using WalletWasabi.Fluent.ViewModels.Navigation;
+using WalletWasabi.Models;
+
+namespace WalletWasabi.Fluent.ViewModels.Wallets.Send
+{
+	[NavigationMetaData(Title = "Advanced")]
+	public partial class AdvancedSendOptionsViewModel : RoutableViewModel
+	{
+		private readonly TransactionInfo _transactionInfo;
+
+		[AutoNotify] private string _customFee;
+
+		public AdvancedSendOptionsViewModel(TransactionInfo transactionInfo)
+		{
+			_transactionInfo = transactionInfo;
+
+			_customFee = transactionInfo.CustomFeeRate != FeeRate.Zero
+				? transactionInfo.CustomFeeRate.SatoshiPerByte.ToString(CultureInfo.InvariantCulture)
+				: "";
+
+			EnableBack = false;
+			SetupCancel(enableCancel: true, enableCancelOnEscape: true, enableCancelOnPressed: true);
+
+			this.ValidateProperty(x => x.CustomFee, ValidateCustomFee);
+
+			var nextCommandCanExecute =
+				this.WhenAnyValue(x => x.CustomFee)
+					.Select(_ =>
+					{
+						var noError = !Validations.Any;
+						var somethingFilled = CustomFee is not null or "";
```

Comment:
```suggestion
						var somethingFilled = string.IsNullOrWhiteSpace(CustomFee);
```

Reply:
This is not correct. Your suggestions should be negated, but can you explain why would it be better?

Reply:
You are correct about the negate and i thought we allow whitespace as a valid input, but it's handled.
Don't mind the suggestions.

Reply:
@soosr I guess @adamPetho is right about this one.
It fixes https://github.com/zkSNACKs/WalletWasabi/pull/6617#issuecomment-981112022

Note that we can't go back from using a custom fee rate if we don't allow "".

Reply:
> Note that we can't go back from using a custom fee rate if we don't allow "".

That is the point. I fixed https://github.com/zkSNACKs/WalletWasabi/pull/6617#issuecomment-981112022.